src/app/api/run/[runId]/delete/route.ts

```diff
@@ -0,0 +1,33 @@
+import { NextRequest, NextResponse } from 'next/server'
```

Comment:
Is this just for development? I don't think we have a use-case for deleting results via API (only when results are approved & sent to the management app)